cloud-sql/mysql/mysql/server.js

```diff
@@ -83,19 +83,24 @@ const pool = mysql.createPool({
   // [END cloud_sql_mysql_mysql_backoff]
 
   //[END_EXCLUDE]
-});
+};
+let pool;
+(async () => {
```

Comment:
nit: I spoke with Ben about the best way to handle top level awaits, he mentioned this might be preferable as a named function:
```
(async function startPool () {
   ...
})()
```

Reply:
Yes, please name the function. In the sample template we don't immediately invoke the function but use the function name instead 

```
async function startPool() {
   // ...
};
startPool();
```

Reply:
`async function startPool() {}` triggered a linting error (`Expected a function expression  func-style`).

Proceeding with the same guidance using `const createPool = async () => {}`.